### DISCUSSION TITLE
Patching image for Pixel 7 pro

### DISCUSSION THREAD

**Discussion by neo8848** [tiann/KernelSU#1055] [General]
Hi, I tried kernelSU by installing it using kernel flasher (phone was rooted with magisk already). Now, I have updated the phone and I do not have the root.  I want to install kernelSU using android kitchen method. Question is do i need to patch init_boot.img file for kernel  instead of stock boot.img ? because for magisk I had to patch init_boot and flash it. Is it same for kernelSU?

**Comment by tiann** [comment]
No, KernelSU resides in kernel, so it always patch boot.img

**Reply by neo8848** [reply]
I tried the method but phone failed to boot and falls back to fastboot. the stock image size is 64Mb with 24 Mb boot.img-kernel ; when replaced with 42Mb (anyrecovery image) final image size is only 42Mb. Am i missing something or is this expected behavior? Is there any other way I should be looking to root my pixel 7 pro android 14?

**Reply by neo8848** [reply]
edit: size has nothing to do with it ; but phone is not bootable with patched image from 
AnyKernel3-android13-5.10.157_2023-03.zip .
